lib/full-site-editing/full-site-editing.php

```diff
@@ -14,6 +14,18 @@ function gutenberg_is_fse_theme() {
 	return is_readable( get_stylesheet_directory() . '/block-templates/index.html' );
 }
 
+/**
+ * Returns whether the current theme is FSE-enabled or not.
+ *
+ * @return boolean Whether the current theme is FSE-enabled or not.
+ */
+function gutenberg_is_fse_enabled() {
+	return apply_filters(
+		'full_site_editing_enabled',
```

Comment:
I'm not sure we need that filter, plugins can still enable theme supports. this seems like a duplication.

Reply:
You're right. Removed 👍